.github/mergeable.yml

```diff
@@ -0,0 +1,28 @@
+# ProBot Mergeable Bot
+# https://github.com/jusx/mergeable
+
+mergeable:
+  pull_requests:
+    approvals:
+      # Minimum of approvals needed.
+      min: 1
+        message: 'The PR must have a minimum of 1 approvals.'
```

Comment:
This can be done with GitHub I believe

Reply:
Maybe. Although, if we are going to use mergeable, I prefer to have all the configs in the same place.